liquidctl.8

```diff
@@ -195,6 +207,37 @@ Internal data used by some drivers.
 .
 .SH DEVICE SPECIFICS
 .
+.SS Corsair Commander Pro
+.SS Corsair Lighting Node Pro
+Cooling channels: \fIfan\fR, \fIfan[1\-5]\fR. (Commander Pro only)
```

Comment:
I missed this before: what does `fan` mean on a Commander Pro?

It it syncs all fans, then we have a convention of calling that (pseudo) channel "sync".

Reply:
hmm. Some of the drivers, (ie the H100i uses the `fan` channel to control both fans. I can change this to sync if you'd prefer.